src/main/resources/init_db.sql

```diff
@@ -23,3 +23,67 @@ ALTER TABLE `taxi_service`.`manufacturer`
     CHANGE COLUMN `nameManufacturer` `name_manufacturer` VARCHAR(225) NOT NULL ,
     CHANGE COLUMN `countryManufacturer` `country_manufacturer` VARCHAR(225) NOT NULL ,
     CHANGE COLUMN `deleteManufacturer` `delete_manufacturer` TINYINT NOT NULL DEFAULT '0' ;
+
+CREATE TABLE `taxi_service`.`drivers` (
+                                          `id_driver` BIGINT NOT NULL,
+                                          `name_driver` VARCHAR(225) NOT NULL,
+                                          `licence_number_driver` VARCHAR(225) NOT NULL,
+                                          `delete_driver` TINYINT NOT NULL DEFAULT 0,
```

Comment:
```suggestion
                                          `id` BIGINT NOT NULL,
                                          `name` VARCHAR(225) NOT NULL,
                                          `licence` VARCHAR(225) NOT NULL,
                                          `delete` TINYINT NOT NULL DEFAULT 0,
```
same for `manufacturer` and `car` tables - don't repeat the table name in columns